ush/config_defaults.yaml

```diff
@@ -1692,10 +1699,57 @@ task_make_ics:
   FVCOM_DIR: ""
   FVCOM_FILE: "fvcom.nc"
   VCOORD_FILE: "{{ workflow.FIXam }}/global_hyblev.l65.txt"
-
-#----------------------------
-# MAKE LBCS config parameters
+  #------------------------------------------------------------------------
+  input_files_metadata_path: '{{ task_get_extrn_ics.rundir }}/{{ workflow.EXTRN_MDL_VAR_DEFNS_FN }}.yaml'
+  chgres_cube:
+    execution:
+      executable: "{{ user.EXECdir }}/chgres_cube"
+      envcmds:
+        - module use {{ user.HOMEdir }}/modulefiles
+        - module load build_{{ user.MACHINE|lower }}_{{ workflow.COMPILER }}
+      mpiargs:
+        - "--ntasks $SLURM_CPUS_ON_NODE"
+      mpicmd: '{{ platform.BATCH_RUN_CMD }}'
+      threads: 1
+    namelist:
+      update_values:
+        config:
+          atm_files_input_grid: "{{ 'fn_atm' | env }}"
+          convert_atm: true
+          convert_nst: true
+          convert_sfc: true
+          cycle_day: !int "{{ cycle.strftime('%d') }}"
+          cycle_hour: !int "{{ cycle.strftime('%H') }}"
+          cycle_mon: !int "{{ cycle.strftime('%m') }}"
+          data_dir_input_grid: '{{ task_get_extrn_ics.rundir }}'
```

Comment:
Wondering which references of `EXTRN` or `extrn` we can update? Are we able to update this to a better name or is this in a different process to be updated down the line?

Reply:
I've been wary of opening more cans of worms by expanding to all of `task_get_extrn_ics`/`lbcs`, I'm glad I was able to get away with that YAML change as it is